app/src/app/utils/api/apiHandlers/getDocument.ts

```diff
@@ -3,16 +3,16 @@ import {DocumentObject} from './types';
 import {useMapStore} from '@store/mapStore';
 
 export const getDocument = async (document_id: string): Promise<DocumentObject> => {
+  let url = `${process.env.NEXT_PUBLIC_API_URL}/api/document/${document_id}`;
+
+  if (!document_id) throw new Error('No document id found');
+
   const userID = useMapStore.getState().userID;
-  if (document_id && userID) {
-    return await axios
-      .post(`${process.env.NEXT_PUBLIC_API_URL}/api/document/${document_id}`, {
-        user_id: userID,
-      })
-      .then(res => {
-        return res.data;
-      });
-  } else {
-    throw new Error('No document id found');
+  if (userID) {
+    url += `?user_id=${userID}`;
```

Comment:
H: For better URL safety use `const url = new URL(${process.env...` and `url.searchParams.set('user_id', userID)` + `axios.get(url.toString())`